templates/partials/elevio.mustache

```diff
@@ -0,0 +1,13 @@
+<div id='elevio-shell'>
+    <span class='text'>NEED HELP?</span>
```

Comment:
I don't have translation for `NEED HELP?`. And haven't decided how to implement it. Any suggestions?